doc/release-notes/release-7.sgml.in

```diff
@@ -39,23 +38,28 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 <sect1>Cache Manager changes<label id="mgr">
 <p>For more information about the Cache Manager feature, see <url url="https://wiki.squid-cache.org/Features/CacheManager/Index" name="wiki">.
 
-<p>
-<descrip>
-	<tag>cache_object URI</tag>
-	<p>Cache manager is no longer accessible by URIs with cache_object scheme.
+<p>In order to reduce workload on the Squid development team we have chosen to stop
+   providing several tools related to Cache Manager which have previously been
+   bundled with Squid.
 
-    <tag>cachemgr</tag>
-    <p>The 'cachemgr' tool is no longer supported and distributed with squid.
-    A plethora of tools, such as curl, wget, or any web browser,
-    can be used in its place
+<sect2>Removal of the <em>squidclient</em> tool.
+<p>Popular command-line tools such as <em>curl</em> or <em>wget</em> 
```

Comment:
```
running: git -c core.whitespace=-blank-at-eof diff --check refs/pull/1542/merge^1
  doc/release-notes/release-7.sgml.in:46: trailing whitespace.
  +<p>Popular command-line tools such as <em>curl</em> or <em>wget</em> 
  Error:  git diff detected bad whitespace changes.
```

The suggestion below should address the above problem detected by [CI](https://github.com/squid-cache/squid/actions/runs/6646976944/job/18061486478?pr=1542).


```suggestion
<p>Popular command-line tools such as <em>curl</em> or <em>wget</em>
```